core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -709,7 +709,7 @@ write_files:
 
       ETCD_INITIAL_CLUSTER_STATE=new
       ETCD_DATA_DIR=/var/lib/etcd2
-      ETCD_LISTEN_CLIENT_URLS=https://$private_ip:2379
+      ETCD_LISTEN_CLIENT_URLS=https://0.0.0.0:2379
```

Comment:
This is required in order for migration process (etcdadm) running on the new etcd servers to connect to the existing cluster and export all kubernetes keys.  The migration works around this export and import between the etcd clusters.

Reply:
Awesome trick!